src/utils/processTrigger.ts

```diff
@@ -51,6 +60,9 @@ export function processTrigger(
             reload(iFrame);
             return res(null);
         }
+        if (messageType === HostEvent.GetIframeSrc) {
+            return getIframeSrc(iFrame);
```

Comment:
pass the result into res check above

also test it out in app embed , if route changes return proper urls

Reply:
yeah, right. 
Will test in app embed